class AssertionErrors:
    URL_NOT_FOUND = 'Expected:{}, but we got:{}'
